tests/benchmarks/doc.go

```diff
@@ -14,5 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package benchmarks contains go-benchmark tests for Klaytn performance.
+/*
+Package benchmarks contains go-benchmark tests for Klaytn performance.
```

Comment:
```suggestion
Package benchmarks contains go-benchmark tests to measure performance of Klaytn network.
```

Reply:
How about changing "Klaytn network" to "Klaytn"?
`Package benchmarks contains go-benchmark tests to measure performance of Klaytn`

Reply:
Sounds good. :)